operator/pkg/ciliumidentity/cell.go

```diff
@@ -41,4 +41,7 @@ type SharedConfig struct {
 	// EnableCiliumEndpointSlice indicates if the Cilium Endpoint Slice feature is
 	// enabled.
 	EnableCiliumEndpointSlice bool
+	// EnableNetworkPolicySystem indicates if the network policy enforcement
+	// system is enabled for K8s, Cilium and Cilium Clusterwide network policies.
+	EnableNetworkPolicySystem bool
```

Comment:
`EnableNetworkPolicy` or `EnableNetworkPolicyEnforcement` - the "system" seems redundant.

Also, shouldn't this be `DisableNetworkPolicy`? Default value for a `bool` in go is `false`, and I'm a bit afraid that it's too easy to "accidentially" disable this somehow